### ISSUE TITLE
Lab 9 peer review - Paolo Magliano s314867

### ISSUE THREAD

**Issue by paolo-magliano** [andrea-ga/computational-intelligence#4]
The project is really complete and thorough, the code is structured well and the documentation shows clearly the results and the settings with graphs.

The implementation of the Evolutionary Algorithm tests many possible variation of the canonical methods such as mutation and crossover methods. There is also different ways to generate the new population using the elitism concept.

The idea of evaluate the population beyond their fitness with a sort of distance metric could be a usefull to promote heterogeneity between individual and to try to don't get stuck in a local maximum.

Summing up, the project hits the goal to discover and test multiple options to find the most suitable for this problem.
